File: 151004/Matsiushyn/Lab3/discussion/DC_Lab1/DTO/PostResponseTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class PostResponseTo(int Id, int? tweetId, string? Content) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public int? tweetId { get; set; } = tweetId;

        public string? Content { get; set; } = Content;

    }
}

File: 151001/Baran/Lab1/src/main/java/by/bsuir/dto/PostResponseTo.java
package by.bsuir.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PostResponseTo {
    private Long id;
    private String content;
    private Long issueId;
}
